### PR TITLE
Continuous Integration

### LINKED ISSUE #19: Add CI and Automate Testing
Let's integrate the CI and automated testing

### PR DESCRIPTION
**Pull request by stanleynguyen**
- Add integration tests
- Github Actions for:
  - unit & integration test
  - linting using `go vet` & `golangci_lint`
  - coveralls

Resolves #19

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go-version }}
+      - run: go mod download
+      - run: go test ./...
+      - run: go run ./integration_test
+        env:
+          SECRET_KEY: xnd_development_chBDpzjQZEI8nncBLrWtwau5r7rgiuunCJ4JCqsd68rXemDd74BnRpdO0bIZAMK
```
Comment (kevindavee) [inline .github/workflows/test.yml]: Can this key stored not in the `.yml` file?

Comment (stanleynguyen) [reply .github/workflows/test.yml]: can but it would require admin to add the secret, too much of a hassle imo since this is just a development key

Comment (kevindavee) [reply .github/workflows/test.yml]: Alright

**Comment by kevindavee** [comment]
LGTM

**Comment by stanleynguyen** [comment]
Not working across forks currently https://github.com/xendit/xendit-go/pull/32

**Comment by stanleynguyen** [comment]
Made some changes to serve peeps who still use `$GOPATH`

**Pull request merged**